### ISSUE TITLE
dw735 connector does not fit onto planer outlet

### ISSUE THREAD

**Issue by luisillo26** [ostat/vacuum-hose-adapter-openscad#8]
As commented in printables, the dw735 connector don't fit onto the planer outlet.

I printed the test ring but unfortunately the fixed pin on the planer does not slide nicely to the bottom of any of the 4 "hooks" on the adapter so it gets stuck without letting the metal latch to engage on the other side. It looks like there is not enough clearance.

![IMG_4528](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/1028540/9dc95a8b-136e-44b6-92f1-24b5f017a909)

**Comment by ostat** [comment]
Can you include a photo of the hole the pin should go into, interested to see how close the pin is to the hole?

**Comment by luisillo26** [comment]
I added that photo in the PR

**Comment by ostat** [comment]
Thanks. 

I am surprised that shortening the pin fixed the issue. I expected that you would need to make the pin longer, to increase fixedPinLength for more hole depth for the pin, or adjust one of the other slot variables. 
  

Was the issue that the fixed pin got stuck inside the slot, or was it the walls around the pin hole was hitting the plastic on the planer preventing it seating?

**Comment by luisillo26** [comment]
Yeah, the slots walls prevented the adapter to fully seat because they were rubbing against the yellow plastic but making the slots slimmer fixed the issue. I'm printing the full version now, will update with the final picture

**Comment by ostat** [comment]
This is what mine looks like. Not the best filament colour choice. 

![image](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/2128234/d74bb267-8bff-4e2d-8a8f-4d325d2bea96)

**Comment by luisillo26** [comment]
That dust port looks a few millimeters longer than my planer, I guess that's why it didn't fit with the 1st test print

**Comment by ostat** [comment]
Agreed, also you can see in mine that the lip under the sticker looks much bigger than yours. So there has been a revision in the design somewhere.

Do you think it would be worth doing something like this?

![image](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/2128234/e09ea50c-da8f-4df6-8c85-d472452fbac5)

**Comment by ostat** [comment]
re-opened to continue discussion, post accept of merge.

**Comment by luisillo26** [comment]
I think either that option or having the option to completely exclude slot "covers" in [these lines](https://github.com/ostat/vacuum-hose-adapter-openscad/blob/4bec6f7580d9dfa97d7f7fa6cea9fd80005ea4a7/vacuum-hose-adapter-modules.scad#L1120C11-L1137C14) by using another parameter/variable.

I printed it without the covers like the factory one, just like you said and it's held very well.

![IMG_4541](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/1028540/dcd595ad-0caa-43c1-968e-92588fd5aecf)

![IMG_4542](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/1028540/4738da4a-5bc8-48a7-9634-2fd2ab4647a5)

**Comment by luisillo26** [comment]
Here is the full print with the changes from the merge, it printed away from perfect because I used supports with PETG which sucks but it's totally functional. The only thing I noticed is it's a little wobbly, it rocks up and down unlike the factory adapter, it must be because that one has a stop of 90 degrees from the internal wall where the slots are placed instead of the tapered one in you design.

![IMG_4539](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/1028540/f43e94bb-7a55-4470-a845-c99960defef4)

![IMG_4540](https://github.com/ostat/vacuum-hose-adapter-openscad/assets/1028540/ac719f4e-1666-414a-a56e-a82266fc1f00)

**Comment by ostat** [comment]
I think this can be closed, If someone has further issues we can re-open
